gaffer-worker/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>uk.gov.gchq.gaffer</groupId>
+    <artifactId>gaffer-experimental</artifactId>
+    <version>1.13.0</version>
+  </parent>
+
+  <artifactId>gaffer-worker</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>gaffer-worker</name>
```

Comment:
To avoid confusion with the controller's worker, I'd call this module something like gaas-rest

Reply:
Done

Reply:
Not seeing the commit come through. Have you pushed?

Reply:
Not yet we are finishing up all fixes first before pushing

Reply:
Done, renamed to gaas-rest